src/interface_adapter/round/RoundState.java

```diff
@@ -10,6 +13,13 @@ public class RoundState {
     private int currentLives;
     private int maxRounds;
     private int currentRoundNumber;
+    private String multipleChoice1 = "";
+    private String multipleChoice2 = "";
+    private String multipleChoice3 = "";
+    private String multipleChoice4 = "";
```

Comment:
```suggestion
    private List<String> multipleChoiceOptions;
```
One small note here (and on others like this): it is better practice to use the interface instead of concrete implementations for variable/method declarations since there is nothing particularly special about ArrayList in these cases.